swig/python/flamegpu.i

```diff
@@ -524,7 +526,14 @@ class ModelVis;
 
 // Must wrap these prior to HostAPI where they are used to avoid issues with no default constructors etc.
 %include "flamegpu/runtime/utility/HostRandom.cuh"
+
+//%ignore flamegpu::HostEnvironment::getMacroProperty();
```

Comment:
If these commented out ignore/renames are not required could they be removed? Or atleast explained why they are still here?

Reply:
I'll remove, was probably dev junk from trying to get it to work